src/modules/config.example.ts

```diff
@@ -5,6 +5,12 @@ export default {
     owner: '', // Owner prefix (p@ on Penny)
   },
 
+  owners: [
```

Comment:
For the sake of consistency I'd prefer to leave these blank since nothing else is filled in